src/avfoundation.cs

```diff
@@ -11274,6 +11533,14 @@ interface AVSampleBufferDisplayLayer {
 		[Field ("AVSampleBufferDisplayLayerFailedToDecodeNotificationErrorKey")]
 		NSString FailedToDecodeNotificationErrorKey { get; }
 
+		[TV (11, 0), NoWatch, Mac (10, 13), iOS (11, 0)]
+		[Field ("AVSampleBufferAudioRendererWasFlushedAutomaticallyNotification")]
+		NSString AudioRendererWasFlushedAutomaticallyNotification { get; }
+
+		[TV (11, 0), NoWatch, Mac (10, 13), iOS (11, 0)]
+		[Field ("AVSampleBufferAudioRendererFlushTimeKey")]
```

Comment:
Is this part of the Flushed Automatically Notification payload?

Reply:
Yes, it is part of the AVSampleBufferAudioRendererWasFlushedAutomaticallyNotification, as per the headers:

 The renderer may flush enqueued media data when the user routes playback to a new destination.  The renderer may also flush enqueued media data when the playback rat
e of the attached AVSampleBufferRenderSynchronizer is changed (e.g. 1.0 -> 2.0 or 1.0 -> 0.0 -> 2.0), however no flush will occur for normal pauses (non-zero -> 0.0) and resumes (0.
0 -> same non-zero rate as before).

                When an automatic flush occurs, the attached render synchronizer's timebase will remain running at its current rate.  It is typically best to respond to this notification by enqueueing media data with timestamps starting at the timebase's current time.  To the listener, this will sound similar to muting the audio for a short period of time.  If it is more desirable to ensure that all audio is played than to keep the timeline moving, you may also stop the synchronizer, set the synchronizer's current time to the value of AVSampleBufferAudioRendererFlushTimeKey, start reenqueueing sample buffers with timestamps starting at that time, and restart the synchronizer.  To the listener, this will sound similar to pausing the audio for a short period of time.

                This notification is delivered on an arbitrary thread.  If sample buffers are being enqueued with the renderer concurrently with the receipt of this notification, it is possible that one or more sample buffers will remain enqueued in the renderer.  This is generally undesirable, because the sample buffers that remain will likely have timestamps far ahead of the timebase's current time and so won't be rendered for some time.  The best practice is to invoke the -flush method, in a manner that is serialized with enqueueing sample buffers, after receiving this notification and before resuming the enqueueing of sample buffers.

Reply:
Then you need to add it to the Notification attribute as described here https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#NotificationAttribute

look for this

> If your notification contains parameters, you should specify a helper EventArgs interface, like this: